.gitignore

```diff
@@ -1,3 +1,4 @@
 vendor/
 .php_cs.cache
 composer.lock
+.idea
```

Comment:
should be reverted. This is not related to the project ignore rules, but to your own setup. So this should be ignored as part of your setup instead (see https://help.github.com/en/github/using-git/ignoring-files)

Reply:
Oh yes, fixed.